daily-builds.md

```diff
@@ -1,23 +1,23 @@
-# .NET Core Daily Builds -- Servicing and future releases
+# .NET Daily Builds -- Servicing and future releases
 
-The .NET Core team builds many branches of the product daily. You can install these builds to try out functionality prior to release.
+The .NET team builds many branches of the product daily. You can install these builds to try out functionality prior to release.
 
 ## Future Releases
 
-Preview branches are for new major/minor versions of .NET Core that have not yet been released and are not yet supported.
+Preview branches are for new versions of .NET that have not yet been released and are not yet supported.
 
-|Component|*.NET 5.0*
+|Component|*.NET 6.0*
 |:------:|:------:|
-|SDK|[.NET Core SDK 5.0.xxx  (Master)](https://github.com/dotnet/core-sdk/blob/master/README.md#installers-and-binaries)
-|ASP.NET Core|[ASP.NET Core Shared Framework 5.0 (Master)](https://github.com/aspnet/AspNetCore/blob/master/docs/DailyBuilds.md)
-|Runtime|[.NET Core Runtime 5.0 (Master)](https://github.com/dotnet/core-setup/blob/master/README.md#daily-builds)
+|SDK|[.NET SDK 6.0.xxx](https://github.com/dotnet/installer/blob/master/README.md#installers-and-binaries)
+|ASP.NET Core|[ASP.NET Core 6.0](https://github.com/aspnet/AspNetCore/blob/master/docs/DailyBuilds.md)
+|Runtime|[.NET Runtime 6.0](https://github.com/dotnet/installer/blob/master/README.md#daily-builds)
 
 ## Servicing Releases
 
-Servicing branches are for new patch versions of .NET Core that have not yet been released and are not yet supported. These patch versions are for previously released major/minor versions.
+Servicing branches are for new patch versions of .NET that have not yet been released and are not yet supported.
 
-|Component|*.NET Core 3.1*|*.NET Core 2.1*
-|:------:|:------:|:------:|
-|SDK|[.NET Core SDK 3.1](https://github.com/dotnet/core-sdk/blob/master/README.md#installers-and-binaries)|[.NET Core SDK 2.1.xxx](https://github.com/dotnet/cli/blob/release/2.1/README.md#installers-and-binaries)|
-|ASP.NET Core|[ASP.NET Core Shared Framework 3.1](https://github.com/aspnet/AspNetCore/blob/master/docs/DailyBuilds.md)|[ASP.NET Core Shared Framework 2.1](https://github.com/aspnet/AspNetCore/blob/master/docs/DailyBuilds.md)|
-|Runtime|[.NET Core Runtime 3.1](https://github.com/dotnet/core-setup/blob/master/README.md#daily-builds)|[.NET Core Runtime 2.1](https://github.com/dotnet/core-setup/blob/master/README.md#daily-builds)|
\ No newline at end of file
+|Component|*.NET 5.0*|*.NET Core 3.1*|*.NET Core 2.1*|
+|:------:|:------:|:------:|:------:|
+|SDK|[.NET SDK 5.0.xxx](https://github.com/dotnet/installer/blob/master/README.md#installers-and-binaries)|[.NET Core SDK 3.1](https://github.com/dotnet/core-sdk/blob/master/README.md#installers-and-binaries)|[.NET Core SDK 2.1.xxx](https://github.com/dotnet/cli/blob/release/2.1/README.md#installers-and-binaries)|
```

Comment:
SDK version should be '3.1.xxx' to be consistent with 2.1 and 5.0 column representations?